styles.css

```diff
@@ -39,3 +39,19 @@ img {
     background: red;
   }
 }
+h1 {
+  margin : 0;
+
+}
+h3 {
+  margin: 0;
+
+}
+p {
+  margin: 0;
+
+}
+h2 {
+  margin: 0;
+
+}
```

Comment:
The answer to this question requires using CSS' direct child syntax. So it would look something like this...

```css
header, footer, .main, aside > * {
  margin: 0;
}
```
> `>` means direct child, `*` means everything. Together they mean "select all direct children".